src/Native/Runtime/CMakeLists.txt

```diff
@@ -78,13 +78,15 @@ if(WIN32)
     windows/PalRedhawkMinWin.cpp
   )
 
-  list(APPEND FULL_RUNTIME_SOURCES 
-    windows/CoffNativeCodeManager.cpp
```

Comment:
This file would be needed to make things actually work. It would be better to ifdef out the parts that do not build in it for x86 with TODOs.